packages/backend/discovery/espresso/ethereum/config.jsonc

```diff
@@ -18,7 +19,29 @@
         "stateHistoryFirstIndex",
         "getStateHistoryCount",
         "finalizedState"
-      ]
+      ],
+      "fields": {
+        "permissionedProver": {
+          "target": {
+            "permissions": [
+              {
+                "type": "configure",
+                "description": "relayer (prover) address that is allowed to call newFinalizedState() to prove the latest HotShot state to the Light Client contract on Ethereum."
```

Comment:
This description doesn't look well on frontend because it looks like this:

```
* EOA 1

Can change the configuration of HotShotLightClient - relayer ([prover](http://localhost:3000/glossary#prover)) address that is allowed to call newFinalizedState() to prove the latest HotShot state to the Light [Client](http://localhost:3000/glossary#client) contract on Ethereum.
```

I suggest to change description like this, so it reads better: "can call newFinalizedState() to prove the latest HotShot state"

Reply:
https://github.com/l2beat/l2beat/pull/6007/commits/a12a9a480b1c6c4649afec9121db3f8c6fdc65dd